src/Komponents/PrivateRoutes/Routes/PrivateRoute.jsx

```diff
@@ -0,0 +1,13 @@
+import MainPage from "../../../pages/MainPage/MainPage";
+
+import { CUR_USER_TOKEN } from "../../../redux/store/usersReducer";
+
+import { Navigate } from "react-router";
+
+
+const Private = () => {
+  const token = localStorage.getItem(CUR_USER_TOKEN);
```

Comment:
u can check if user exist by that token and then do some actions

Reply:
what do you mean ?
i already create this logic in log in